homeassistant/components/songpal/config_flow.py

```diff
@@ -0,0 +1,158 @@
+"""Config flow to configure songpal component."""
+import logging
+from typing import Optional
+from urllib.parse import urlparse
+
+from songpal import Device
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.components import ssdp
+from homeassistant.const import CONF_HOST, CONF_NAME
+
+from .const import CONF_ENDPOINT, CONF_MODEL, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+DEFAULT_NAME = "Songpal"
```

Comment:
```suggestion
DEFAULT_NAME = "Songpal device"
```
maybe? If this is even needed?